src/me/mrCookieSlime/Slimefun/Android/ProgrammableAndroid.java

```diff
@@ -215,6 +216,8 @@ public void onPlace(Player p, Block b, SlimefunItem item) {
 			public boolean onBreak(Player p, Block b, SlimefunItem item, UnregisterReason reason) {
 				boolean allow =  reason.equals(UnregisterReason.PLAYER_BREAK) && (BlockStorage.getLocationInfo(b.getLocation(), "owner").equals(p.getUniqueId().toString()) || p.hasPermission("slimefun.android.bypass"));
 
+				if (BlockStorage.check(b) != null) return false;
```

Comment:
You should rather use BlockStorage.hasBlockInfo() for this.

Reply:
So it would be 

if (BlockStorage.hasBlockInfo() != null) return false;

Correct? Or would it be this: if (BlockStorage.hasBlockInfo(b)) return false;?

I just woke up, so my apologies if I am incorrect

Reply:
It returns a boolean, so a null-check would make no sense.